float32 joint_1
float32 joint_2
float32 joint_3

int8 speed
